src/login/logind-dbus.c

```diff
@@ -2945,6 +2946,16 @@ int manager_start_scope(
                         return r;
         }
 
+        if (!isempty(requires)) {
+                r = sd_bus_message_append(m, "(sv)", "After", "as", 1, requires);
+                if (r < 0)
+                        return r;
+
+                r = sd_bus_message_append(m, "(sv)", "Requires", "as", 1, requires);
+                if (r < 0)
+                        return r;
+        }
```

Comment:
not that it matters, but you could merge these two s_b_m_a() ines into one, the format string logic accepts that.